lte/gateway/python/magma/pipelined/qos/common.py

```diff
@@ -32,6 +32,7 @@
     get_subscriber_key,
 )
 from magma.pipelined.qos.utils import QosStore
+from redis import ConnectionError
```

Comment:
This import changes which exception is caught in the existing `except ConnectionError:` block of `_is_redis_available`.